client/components/tooltip/index.jsx

```diff
@@ -4,69 +4,63 @@
  * External dependencies
  */
 
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Popover from 'components/popover';
-import { isMobile } from 'lib/viewport';
+import { useMobileBreakpoint } from 'lib/viewport/react-helpers';
```

Comment:
I think that `lib/viewport/react` would be a more pleasant name for the module. The `helpers` bit doesn't convey much information.

Reply:
Renamed.